x/staking/Cargo.toml

```diff
@@ -18,3 +18,7 @@ ibc-proto = { workspace = true }
 tracing = { workspace = true }
 tonic = { workspace = true }
 tower-layer = { workspace = true }
+serde-aux = { workspace = true }
+
+[dev-dependencies]
+hex = { workspace = true }
```

Comment:
I think that for new deps crate  `data-enoding`  is more preferable.

Reply:
https://docs.rs/data-encoding/latest/data_encoding/

Reply:
Changed to use `data-encoding`.